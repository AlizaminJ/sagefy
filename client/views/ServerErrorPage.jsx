/* eslint-disable jsx-a11y/anchor-is-valid, no-script-url */

import React from 'react'
import { string } from 'prop-types'
import Layout from './components/Layout'
import Icon from './components/Icon'

export default function ServerErrorPage({ hash }) {
  return (
    <Layout hash={hash} page="ServerErrorPage" title="Server error">
      <section>
        <h1>
          I couldn&apos;t make that happen <Icon i="error" s="h1" />
        </h1>
        <p>
          <Icon i="error" /> 500 Server Error <Icon i="error" />{' '}
        </p>
        <p>
          <a href="/">
            Go back <Icon i="home" /> home
          </a>{' '}
          &bull; You can also <Icon i="left" /> go back and retry.
        </p>
      </section>
    </Layout>
  )
}

ServerErrorPage.propTypes = {
  hash: string.isRequired,
}
