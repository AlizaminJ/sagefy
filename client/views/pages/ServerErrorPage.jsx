/* eslint-disable jsx-a11y/anchor-is-valid, no-script-url */

import React from 'react'
import { Link } from 'react-router-dom'
import Layout from '../components/Layout'
import Icon from '../components/Icon'

export default function ServerErrorPage() {
  return (
    <Layout page="ServerErrorPage" title="Server error" description="-">
      <section>
        <h1>
          I couldn&apos;t make that happen <Icon i="error" s="xxl" />
        </h1>
        <p>
          <Icon i="error" /> 500 Server Error <Icon i="error" />{' '}
        </p>
        <p>
          <Link to="/">
            Go back <Icon i="home" /> home
          </Link>{' '}
          &bull; You can also{' '}
          <a href="javascript:history.back()">
            <Icon i="left" /> go back
          </a>{' '}
          and retry.
        </p>
      </section>
    </Layout>
  )
}
