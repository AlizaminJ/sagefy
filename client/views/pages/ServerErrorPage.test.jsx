import ServerErrorPage from './ServerErrorPage'

describe('ServerErrorPage', () => {
  it('should render the ServerErrorPage', () => {
    expect(ServerErrorPage()).toMatchSnapshot()
  })
})
