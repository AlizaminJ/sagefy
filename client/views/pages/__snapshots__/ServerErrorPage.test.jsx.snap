// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ServerErrorPage should render the ServerErrorPage 1`] = `
<Layout
  description="-"
  page="ServerErrorPage"
  title="Server error"
>
  <section>
    <h1>
      I couldn't make that happen 
      <Icon
        i="error"
        s="h1"
      />
    </h1>
    <p>
      <Icon
        i="error"
        s="m"
      />
       500 Server Error 
      <Icon
        i="error"
        s="m"
      />
       
    </p>
    <p>
      <Link
        replace={false}
        to="/"
      >
        Go back 
        <Icon
          i="home"
          s="m"
        />
         home
      </Link>
       
      â€¢ You can also
       
      <a
        href="javascript:history.back()"
      >
        <Icon
          i="left"
          s="m"
        />
         go back
      </a>
       
      and retry.
    </p>
  </section>
</Layout>
`;
